src/wrapper/util.rs

```diff
@@ -204,7 +204,7 @@ struct ScopedFtz {
 impl ScopedFtz {
     fn enable() -> Self {
         cfg_if::cfg_if! {
-            if #[cfg(target_feature = "sse")] {
+            if #[cfg(all(target_feature = "sse", not(miri)))] {
```

Comment:
Probably easier and less error prone to add a `#[cfg(not(miri))]` around the entire `cfg_if!{}` block, change the `Self`s there to early returns, and to just move the current `else` branch to the end of the function so it that is used as a fallback when either those architecture features aren't available or when MIRI is enabled. I can do that later.

Reply:
Ah, good point! That's indeed a better way to do this, I'll change that right away. :slightly_smiling_face: